homeassistant/components/hangouts/.translations/en.json

```diff
@@ -19,7 +19,8 @@
             "user": {
                 "data": {
                     "email": "E-Mail Address",
-                    "password": "Password"
+                    "password": "Password",
+                    "authorization_code": "Authorization Code"
```

Comment:
You need change `hangouts/strings.json`, then `hangouts/.translations/en.json` will be auto-generated by run `script/translations_develop`. See [dev docs](https://developers.home-assistant.io/docs/en/internationalization_backend_localization.html#translation-strings) for details.

I also suggest make it clear this field is totally optional such as
`"Authorization Code (only used in manual authentication)"`